### PR TITLE
Ahodges/doc 3115

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-3115](https://openedx.atlassian.net/browse/DOC-3115)

From feedback, define OLX, particularly in the XBlock guide and the glossary. 

I also noticed that we never define OLX in the OLX guide, and gave some of the introductory statement in index files in the OLX guide a little spiffing up.
### Date Needed

I'd like to build on this for CAPA problem work that I'll be doing week of 18 July 
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Doc team review (sanity check): @srpearce or @pdesjardins  
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Squash commits

### CODE REVIEW

File: en_us/olx/source/index.rst [hunk 1]
```diff
@@ -4,16 +4,15 @@
 edX Open Learning XML Guide
 ############################
 
-This document is intended for those interested in using Open Learning XML (OLX)
-to develop edX courses.
+This guide is intended for those interested in using Open Learning XML (OLX)
+to develop edX courses. Some familiarity with XML is assumed.
```
Comment (pdesjardins) [inline en_us/olx/source/index.rst]: Maybe reword more simply for better global accessibility?

You must understand XML to use this guide. ?

File: en_us/olx/source/problem-xml/index.rst [hunk 2]
```diff
@@ -8,24 +8,25 @@ The format for edX problems is based on the `LON-CAPA XML format`_, although
...
-OLX is designed to allow mixing and matching of inputtypes, responsetypes, and
-hinters. For example, a numerical grader could match 7+-0.1%. Ideally, you
-could use this grader with any inputtype that outputs a number, including a
-text box, equation input, slider, or multiple choice question. In practice,
-this doesn't always work. For example, in the former case, a multiple choice
-question would not give an output in a format a numerical grader could handle.
+OLX is designed to allow mixing and matching of ``inputtypes``,
+``responsetypes``, and ``hinters``. For example, a numerical grader could match
+7+-0.1%. Ideally, you could use this grader with any ``inputtype`` that outputs
+a number, including a text box, equation input, slider, or multiple choice
```
Comment (pdesjardins) [inline en_us/olx/source/problem-xml/index.rst]: outputs a number > returns numbers as its output

File: en_us/olx/source/problem-xml/index.rst [hunk 3]
```diff
@@ -8,24 +8,25 @@ The format for edX problems is based on the `LON-CAPA XML format`_, although
...
+question. In practice, this does not always work. For example, in the former
+case, a multiple choice question would not give an output in a format a
```
Comment (pdesjardins) [inline en_us/olx/source/problem-xml/index.rst]: in the former case > in the example described above

File: en_us/shared/glossary/glossary.rst [hunk 4]
```diff
@@ -58,8 +58,8 @@ A
 
 **Advanced Editor**
 
-  An XML-only editor in a problem component that allows you to create and edit
-  any type of problem. For more information, see
+  An Open Learning XML (OLX) editor in a problem component that allows you to
```
Comment (pdesjardins) [inline en_us/shared/glossary/glossary.rst]: Should this be OLX (open learning XML)? The spelled out version of CAPA is in parentheses below.

File: en_us/shared/glossary/glossary.rst [hunk 5]
```diff
@@ -623,6 +623,14 @@ N
 O
 ****
 
+**Open Learning XML (OLX)**
+
+  OLX is the XML-based standard that is used to build courses on the Open
```
Comment (pdesjardins) [inline en_us/shared/glossary/glossary.rst]: standard > markup language ?

File: en_us/xblock-tutorial/source/overview/examples.rst [hunk 6]
```diff
@@ -39,7 +39,8 @@ component.
   :alt:  The Google Calendar editor in Studio.
   :width: 600
 
-Course teams or developers can also add a Google calendar using OLX.
+Course teams or developers can also add a Google calendar using Open Learning
+XML (OLX).
```
Comment (pdesjardins) [inline en_us/xblock-tutorial/source/overview/examples.rst]: Maybe this should be OLX (open learning XML) because we use OLX typically, and only need to explain it at first.

Also, I noticed several files in this PR where OLX is used without spelling it out in parentheses. I'm assuming that it's not explained because the files are included in sections that have already explained it. Did you take a look at that?

**Comment by pdesjardins** [comment]
I added some comments. 👍

**Comment by lamagnifica** [comment]
@pdesjardins I changed or added "OLX (open learning XML)" pretty much everywhere I could find it. Please take another look.

**Comment by pdesjardins** [comment]
@lamagnifica, I like these changes. 👍 👍
